cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -5,8 +5,31 @@ <h1 class="d-inline"> {{ record.title_additional or record.title }} </h1>
 
     <p>
         {% if record.title_additional %} <span>{{ record.title }}, </span> {% endif %}
-        {% for author in record.authors %} <small>{{ author.name }};</small> {% endfor %}
-        {% if record.collaboration %} <small>{{ record.collaboration.name }}</small> {% endif %}
+        {% for author in record.authors %}
+            {% if author.orcid %}
+                <a href="https://orcid.org/{{ author.orcid }}"><img src="{{ url_for('static', filename='img/orcid.png') }}" height="14" alt=""></a>
+            {% endif %}
+            {{ author.name }}
+            {% if author.affiliation or author.rorids %}
+                 {% set affiliations = author.get('affiliation', "").split(';') %}
+                 {% set rorids = author.get('rorid', "").split(';') %}
```

Comment:
```suggestion
                 {% set ror_ids = author.get('rorid', "").split(';') %}
```

Reply:
We call it `rorid` in the model, so `rorids` is its natural plural